src/commands/Fun/destruct.js

```diff
@@ -0,0 +1,61 @@
+exports.run = function(bot,msg,args) {
+
+    let parsedArgs = bot.utils.parseArgs(args, ['d:','s:']);
```

Comment:
This variable and the `style` variable should probably be `const`s to follow the code style, but I won't shoot you if you don't change this.